sdk/spring/azure-spring-boot-core/CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
+### New Features
+- Support unified properties as below when the service specific properties are not configured.([#22396](https://github.com/Azure/azure-sdk-for-java/issues/22396))
+    ```
+    spring:
+      cloud:
+        azure:
+          authority-host:
+          client-id:
+          client-secret:
+          certificate-path:
+          msi-enabled:
```

Comment:
what does `msi-enabled` mean?

Reply:
Ops, it's for resource provision and only needed in azure-spring-cloud-context. Will remove it from unified properties.